Marlin/src/module/stepper_indirection.h

```diff
@@ -49,7 +49,11 @@
 // TMC26X drivers have STEP/DIR on normal pins, but ENABLE via SPI
 #if ENABLED(HAVE_TMCDRIVER)
   #include <SPI.h>
-  #include <TMC26XStepper.h>
+  #if defined(STM32F7)
+    #include "src/feature/TMC2660.h"
```

Comment:
You shouldn't use absolute paths in local include notation, if it can be assumed that the folder is included in the builds include path use <> notation otherwise use a relative path, (and yes I hate all the ../../../ everywhere too)

As a side note do platformio and arduino ide include the root project dir in the include path by default?